include/taskolib/Sequence.h

```diff
@@ -463,23 +462,27 @@ class Sequence
     ConstReverseIterator rend() const noexcept { return steps_.crend(); }
 
     /**
-     * Sets setup script.
-     *
-     * \param step_setup_script step setup script.
-     */
-    void set_step_setup_script(gul14::string_view step_setup_script);
-
-    /**
-     * Set an error message for the sequence.
+     * Set an optional Error object to describe the outcome of the last sequence
+     * execution.
      *
-     * \param msg  A string explaining why the sequence stopped prematurely; to signalize
-     *             that the sequence finished normally, it should be empty.
+     * \param opt_error  This should be set to nullopt to indicate that the sequence
+     *                   finished normally, or to an Error object describing why and where
+     *                   it stopped prematurely.
      *
      * \note
      * This is usually not a useful call for end users of the library. It is used by the
      * Executor class and by unit tests.
+     *
+     * \see get_error(), set_error()
      */
-    void set_error_message(gul14::string_view msg);
+    void set_error(gul14::optional<Error> opt_error);
```

Comment:
Is this a different `set_error()` than the one referenced in this function's own docs? :thinking: 

![image](https://user-images.githubusercontent.com/16012374/208054342-87e4989e-bcb5-4ab1-b454-36d3d96f7f6b.png)

Reply:
Oops. Will fix.

Reply:
Done.